src/main/resources/jsonToMT103/component.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 Inc. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<component name="jsonToMT103" type="synapse/template" >
+    <subComponents>
+        <component name="jsonToMT103" >
+            <file>json-to-mt103.xml</file>
+            <description>Transform JSON to SWIFT MT103</description>
+        </component>
+    </subComponents>
+</component>
```

Comment:
Add EOF line

Reply:
Added in commit 96d4dc498094daee16277501378ee52fbb484dd1